versions/go.tex

```diff
@@ -591,6 +589,31 @@ \subsection*{Geschäftsordnungsanträge}\label{geschuxe4ftsordnungsantruxe4ge-1}
 Formale Gegenrede bedeutet nur bekanntzugeben, dass man dagegen ist,
 inhaltliche Gegenrede beinhaltet eine Begründung.
 
+\subsubsection*{Teilnamentliche und Geheime Abstimmung}
+
+Normale Abstimmungen auf der ZaPF sind nicht geheim, aber das
+Abstimmungsverhalten wird auch nicht protokolliert und nur anwesende Teilnehmika
+der ZaPF können sehen ob und wie andere Fachschaften abgestimmt haben.
+Die Teilnamentliche Abstimmung soll dazu dienen einen Antrag besonders zu
+unterstreichen indem man ihm mit dem eignen Namen mehr Gewicht verleiht.
+Durch die Bekanntgabe der Intention namentlich abzustimmen können andere Fachschaften
+abschätzen, ob durch die teilnamentliche Abstimmung ihr Abstimmverhalten bestimmbar
+wird und ggf. Gegenrede einlegen. Dies setzt voraus, dass nur Fachschaften namentlich
+Abstimmen, die auch ihre Intention bekanntgegeben haben.
+Durch die Bedenkzeit soll vermieden werden, dass die Teilnahmentliche Abstimmung durch
+unüberlegte Gegenrede verhindert wird.
+Die Redeleitung sollte bei der Durchführung der Abstimmung darauf achten, dass das
+namentliche Abstimmen ein aktiver Vorgang ist (zum Beispiel durch nach vorne laufen
+und Stimmkarte und Stimme abgeben, danach dann “normale” Abstimmung mit den restlichen
+Fachschaften), sodass nur Fachschaften namentlich abstimmen, die dies umbedingt brauchen
+oder wollen. Dies schützt die Fachschaften, die explizit nicht namentlich abstimmen wollen,
+indem die Wahrscheinlichkeit, dass die Anonymität aufgedeckt wird, verringert wird.
+Als mögliche Alternativen gab es in der Vergangenheit namentliche Abstimmungen und es
+wurde in der Vergangenheit der Vorrang des zuerst gestellten GO-Antrages und teilgeheime
+Abstimmungen vorgeschlagen. Namentliche Abstimmung wurde in der Praxis sehr oft durch geheime
+Abstimmung gekontert. Würde nur der erstgestellte GO-Antrag auf namentliche oder
```

Comment:
In der oben verlinkten Seite im ZaPF-Wiki (sicherheitshalber [hier](https://zapf.wiki/Diskussion:WiSe24_AK_GO-%C3%84nderung:_Abstimmungsformen) nochmal der Link) haben @behrmann und ich versucht darzulegen, warum das Trumpfen der geheimen über die namentliche Abstimmung ein Feature und kein Bug ist. Diese Mechanik würde durch diese Änderung auch nicht abgeschafft, denn die teilnahmentliche Abstimmung kann durch eine einzige Gegenrede abgebrochen werden, eine durch eine Person beantragte geheime Abstimmung wird aber so durchgeführt, egal wie viele im Plenum dagegen sind.

Daher wird mir die Intention aus der Änderung aus ihr selbst nicht wirklich klar.